src/routes/PageLayout.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react'
+import { Outlet, useNavigate, useParams } from 'react-router-dom'
+import classes from '../App.module.css'
+
+function PageLayout() {
+    const { store } = useParams()
+    const navigate = useNavigate()
+
+    useEffect(() => {
+        const storeOptions = ['dataStore', 'userDataStore']
```

Comment:
I think this should be a static const variable that's exported from a utilities file and referenced anywhere these are used.  As it is, this doesn't guarantee that a typo couldn't cause a link to go to `userDatastore` instead of `userDataStore`, for example.

Something like this, maybe...

```js
const DATASTORE = 'dataStore';
const USERDATASTORE = 'userDataStore';
const DATASTORE_OPTIONS = [DATASTORE, USERDATASTORE];
```